<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="TATRC-17323" ignored="0" name="Validate Errors on Login Page" tests="4" failures="4" timestamp="17 Jul 2018 16:30:44 GMT" time="45.818" errors="0">
  <testcase name="@AfterClass afterClass" time="0.001" classname="testCases.BaseTestScriptConfig">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Validating Login Page Title expected [Log in] but found [Log in.],
	Validating password error expected [Invalid login attempt.] but found []">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Validating Login Page Title expected [Log in] but found [Log in.],
Validating password error expected [Invalid login attempt.] but found []
at utilities.AutomationHelper.finishTest(AutomationHelper.java:494)
at testCases.BaseTestScriptConfig.afterClass(BaseTestScriptConfig.java:24)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="execute" time="7.175" classname="testCases.ValidateErrorLoginTypes">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Validating Login Page Title expected [Log in] but found [Log in.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Validating Login Page Title expected [Log in] but found [Log in.]
at testCases.ValidateErrorLoginTypes.execute(ValidateErrorLoginTypes.java:47)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- execute -->
  <testcase name="execute" time="15.105" classname="testCases.ValidateErrorLoginTypes">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Validating Login Page Title expected [Log in] but found [Log in.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Validating Login Page Title expected [Log in] but found [Log in.]
at testCases.ValidateErrorLoginTypes.execute(ValidateErrorLoginTypes.java:47)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- execute -->
  <testcase name="execute" time="6.516" classname="testCases.ValidateErrorLoginTypes">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Validating Login Page Title expected [Log in] but found [Log in.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Validating Login Page Title expected [Log in] but found [Log in.]
at testCases.ValidateErrorLoginTypes.execute(ValidateErrorLoginTypes.java:47)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- execute -->
  <testcase name="execute" time="4.963" classname="testCases.ValidateErrorLoginTypes">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Validating Login Page Title expected [Log in] but found [Log in.],
	Validating password error expected [Invalid login attempt.] but found []">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Validating Login Page Title expected [Log in] but found [Log in.],
Validating password error expected [Invalid login attempt.] but found []
at testCases.ValidateErrorLoginTypes.execute(ValidateErrorLoginTypes.java:47)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- execute -->
</testsuite> <!-- Validate Errors on Login Page -->
